docs/performance/transformers-optimization.md

```diff
@@ -0,0 +1,256 @@
+---
+title: Transformers Optimizer 
+parent: Performance
+nav_order: 4
+---
+# Transformers Optimizer
+{: .no_toc }
+
+## Contents
+{: .no_toc }
+
+* TOC placeholder
+{:toc}
+
+## Transformer Model Optimization Tool Overview
+
+ONNX Runtime automatically applies most optimizations while loading a transformer model. Some of the latest optimizations that have not yet been integrated into ONNX Runtime are available using a [separate tool](https://github.com/microsoft/onnxruntime/tree/main/onnxruntime/python/tools/transformers) that tunes models for the best performance.
```

Comment:
Clarify that the tool is run offline?

Reply:
It's not super clear that this page is about this tool

Reply:
Updated